utils/lib/rr_graph/graph2.py

```diff
@@ -484,6 +484,13 @@ def create_pin_name_from_tile_type_and_pin(
     ):
         return '{}.{}[{}]'.format(tile_type, port_name, pin_idx)
 
+    def create_pin_name_from_tile_type_sub_tile_num_and_pin(
```

Comment:
Needs a docstring

Reply:
Still needs a docstring?  Maybe you forgot to push?

Reply:
Does this need a docstring?  Nothing else in this file has a docstring (including the create_pin_name_from_tile_type_num_and_pin this function is based on).

Reply:
I can write a docstring easily, it just seems weird to only have one function in the file with a docstring.